frontend/views/containers/proposals/RuleRemoveMember.vue

```diff
@@ -19,7 +19,7 @@ proposal-template(
       ref='purpose'
       :placeholder='L("The reason why I\' propositiong this change is...")'
       maxlength='500'
-      :class="{ 'error': $v.form.changeReason.$error }"
+      :class='{ error: $v.form.changeReason.$error }'
```

Comment:
Boy scoutting on line 20 there's `I\'` with a missing `m`

Reply:
I would like to avoid "scouting" in this PR because of the large amount of files changed and the specific purpose of this (linting), as weel to avoid possible conflicts in a parallel PR.